language/move-lang/functional-tests/tests/natives/lcs_large_value.move

```diff
@@ -84,4 +84,4 @@ script {
         M::encode_257();
     }
 }
-// check: ABORTED { code: 453
+// check: ABORTED { code: 453, location: 00000000000000000000000000000001::LCS }
```

Comment:
Just a side note:
```
// check: ABORTED { code: 453, location: 00000000000000000000000000000001::LCS }
// gets desugared into 
// check: ABORTED
// check: {
// check: 453,
// check: location:
// check: 00000000000000000000000000000001::LCS
// check: }
```

To be clear, this was a terrible design choice I made at the time and the whole checker is likely going away soon. For now I'd suggest quoting the whole thing.
```// check: "ABORTED { code: 453, location: 00000000000000000000000000000001::LCS }"```